IPtProxy.go/controller.go

```diff
@@ -77,6 +77,20 @@ import (
 
 const LogFileName = "ipt.log"
 
+const ScrambleSuit = "scramblesuit"
+
+const Obfs2 = "obfs2"
+
+const Obfs3 = "obfs3"
+
+const Obfs4 = "obfs4"
+
+const MeekLite = "meek_lite"
+
+const Webtunnel = "webtunnel"
+
+const Snowflake = "snowflake"
+
```

Comment:
This is a nitpick, and might just be a difference in our coding styles, but I would define and comment these as so:

```golang
// Available transport names
const (
    ScrambleSuit = "scramblesuit"
    Obfs2 = "obfs2"
    Obfs3 = "obfs3"
    Obfs4 = "obfs4"
    MeekLite = "meek_lite"
    Webtunnel = "webtunnel"
    Snowflake = "snowflake"
)
```
See for example how the constants in the `net` package are [defined](https://cs.opensource.google/go/go/+/master:src/net/ip.go;l=22) and [displayed as godoc](https://pkg.go.dev/net#pkg-constants). Anyway, it's up to you, how you want to do this.

Reply:
Ah! Thanks for that hint. I'm not a realy Go aficionado, so I wasn't fully aware of that. Regarding the documentation, I have to think about it ending up in Objective-C header files and Java classes, so I'll keep that. :-) TBH, I find Go slightly underdocumented in certain aspects. (esp. function parameters)